deployer/scripts/upgrade.sh

```diff
@@ -938,8 +962,15 @@ function upgrade_logging() {
   if oc get configmap logging-elasticsearch -o yaml | grep -q MIN_MASTER ; then
      update_es_for_min_masters
   fi
-  scaleUp
 
+  kibanaDC=$(oc get dc logging-kibana -o yaml)
+  kibanaOpsDC=$(oc get dc logging-kibana-ops -o yaml)
+  if [ -z $(echo $kibanaDC | grep -q KIBANA_MEMORY_LIMIT) ] || [ -z $(echo $kibanaDC | grep -q KIBANA_PROXY_MEMORY_LIMIT) ] ||
+     [ -z $(echo $kibanaOpsDC | grep -q KIBANA_MEMORY_LIMIT) ] || [ -z $(echo $kibanaOpsDc | grep -q KIBANA_PROXY_MEMORY_LIMIT) ] ; then
```

Comment:
`grep -q` has no output - so the condition will always be `true`

I suggest this:
```
if echo "$kibanaDC" | grep -q KIBANA_MEMORY_LIMIT && echo "$kibanaDC" | grep -q KIBANA_PROXY_MEMORY_LIMIT && echo "$kibanaOpsDC" | grep -q KIBANA_MEMORY_LIMIT && echo "$kibanaOpsDC" | grep -q KIBANA_PROXY_MEMORY_LIMIT ; then
  echo memory limits already updated
else
  update_kibana_memory_limits
fi
```